### PR TITLE
appgallery-deploy 1.5.4

### PR DESCRIPTION
**Pull request by ferPrieto**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/appgallery-deploy/1.5.4/step.yml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+source_code_url: https://github.com/ferPrieto/steps-app-gallery-deploy
+support_url: https://github.com/ferPrieto/steps-app-gallery-deploy/issues
+published_at: 2021-08-07T13:50:16.273005+01:00
+source:
+  git: https://github.com/ferPrieto/steps-app-gallery-deploy.git
+  commit: ef7f72862c80b8028c3e8ff50c5365b44cc3b232
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
```
Comment (koral--) [inline steps/appgallery-deploy/1.5.4/step.yml:17]: Why not also ionic?

File: steps/appgallery-deploy/1.5.4/step.yml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    is_required: false
+    title: Submit for Review
+    value_options:
+    - "true"
+    - "false"
+  submit_for_review: "true"
+- opts:
+    category: Phase
+    description: Start time of the validity period of the release by phase. The value
+      is UTC time in the following format yyyy-MM-ddTHH:mm:ssZZ. Example:2021-02-17T13:20:15
```
Comment (koral--) [inline steps/appgallery-deploy/1.5.4/step.yml:100]:
```suggestion
      is UTC time in the following format yyyy-MM-ddTHH:mm:ssZZ. Example: 2021-02-17T13:20:15
```

File: steps/appgallery-deploy/1.5.4/step.yml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    category: Phase
+    description: End time of the validity period of the release by phase. The value
+      is UTC time in the following format yyyy-MM-ddTHH:mm:ssZZ. Example:2021-02-20T07:05:15
+    is_expand: true
+    is_required: false
+    title: Phase Release End Time
+  phase_release_end_time: null
+- opts:
+    category: Phase
+    description: Percentage of target users of release by phase.Enter an integer or
```
Comment (koral--) [inline steps/appgallery-deploy/1.5.4/step.yml:115]:
```suggestion
    description: Percentage of target users of release by phase. Enter an integer or
```

File: steps/appgallery-deploy/1.5.4/step.yml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  apt_get:
+  - name: jq
```
Comment (koral--) [inline steps/appgallery-deploy/1.5.4/step.yml:31]: You are using curl in the script so it should be also listed as a dependency.

File: steps/appgallery-deploy/1.5.4/step.yml (hunk 1 above)
Comment (ferPrieto) [reply steps/appgallery-deploy/1.5.4/step.yml:17]: not tested there yet

**Review by ferPrieto — COMMENTED** [review]
Thanks for the heads up @koral-- 👍🏽 

I will definitely include those suggestions in the next PR later next week. Due to the tight frame and the urgency of this PR to be merged I won't make any iteration with those changes.

**Review by lpusok — APPROVED** [review]

**Comment by lpusok** [comment]
@ferPrieto thanks, merged.

**Pull request merged**